src/main/java/uk/sky/cqlmigrate/Lock.java

```diff
@@ -16,6 +16,7 @@
 
     private final LockingMechanism lockingMechanism;
     private final LockConfig lockConfig;
+    private boolean errorHappened;
```

Comment:
ok, I just thought to hide the stuff inside and make it 'lighter' from the top-level, but this can work as well, thanks